pkg/helmreconciler/reconciler.go

```diff
@@ -223,6 +213,23 @@ func (h *HelmReconciler) Delete() error {
 	return err
 }
 
+// allObjectHashes returns a map with object hashes in mm as the keys.
```

Comment:
> in mm as the keys

I don't know what 'mm' means without reading the function body. It looks like its collecting the hashes of all objects across all manifests.

Reply:
That was out of date - fixed.